### ISSUE TITLE
In blazor WASM the httpclient used in the Apexchartservice results in an error when setting the baseadress

### ISSUE THREAD

**Issue by m-lindeman** [apexcharts/Blazor-ApexCharts#556]
I upgraded my Apexcharts nuget to the latest version of 4.0.1 last week and ran into this problem. In my WASM project i have several httpclients i register to use when running my application. It appeard that one of those httpclients was given through dependancy injection to the ApexChartService. This resulted in the error when the service was created, that the client was being used and it was not possible to change the baseurl. The consequence was that i had no charts in my application.

To solve it for me for that moment, i took the project and added it in the solution. I changed the HttpClient to a IHttpClientFactory and this gave me te result i expected. I can create a PR for this in the comming day's, but this is the change i recommend:
Current code:
```
 public ApexChartService(IJSRuntime jSRuntime, HttpClient httpClient, NavigationManager navManager, ApexChartsServiceOptions serviceOptions)
    {
        this.jSRuntime = jSRuntime;
        this.httpClient = httpClient;
       
        httpClient.BaseAddress = new Uri(navManager.BaseUri + "_content/Blazor-ApexCharts/");
```

Proposed code:
```
 public ApexChartService(IJSRuntime jSRuntime, IHttpClientFactory factory, NavigationManager navManager, ApexChartsServiceOptions serviceOptions)
    {
        this.jSRuntime = jSRuntime;
        this.httpClient = factory.CreateClient();
       
        httpClient.BaseAddress = new Uri(navManager.BaseUri + "_content/Blazor-ApexCharts/");
```

**Comment by joadan** [comment]
Hi,

Sorry for this!

I believe the problem is in https://github.com/apexcharts/Blazor-ApexCharts/blob/3515da3bd7d828349fddb7473ed785765593ed82/src/Blazor-ApexCharts/Extensions/ApexChartsExtensions.cs#L30 

the code should be 

`
services.AddHttpClient<IApexChartService>();
`

**Comment by joadan** [comment]
Sorry.. I was wrong.. I updated the code according to your suggestion

**Comment by m-lindeman** [comment]
Thank you fot completing it! I was ill the last week so i was unable to do anything.

**Comment by joadan** [comment]
Fixed in v5.0.0
